src/plugins/MessagesPlugin/DevMessagesRepository.ts

```diff
@@ -17,7 +17,7 @@ const messages: Record<MessageKeys, string> = {
 	'wikibaselexeme-newlexeme-lemma-language-empty-error': 'The spelling variant field is empty; please make an entry.',
 	'wikibaselexeme-newlexeme-lemma-language-invalid-error': 'Spelling variant must be a valid language code.',
 	'wikibaselexeme-newlexeme-lemma-language-placeholder-with-example': 'Language code for the Lemma\'s spelling variant, e.g. \'$1\'',
-	'wikibaselexeme-newlexeme-lemma-language-help-link-target': 'https://www.wikidata.org/wiki/Special:MyLanguage/Help:Monolingual_text_languages',
+	'wikibaselexeme-newlexeme-lemma-language-help-link-target': 'https://www.wikidata.org/wiki/Wikidata:Lexicographical_data/Spelling_variant',
```

Comment:
We probably still want to have Special:MyLanguage on the new link (I assume the page is going to be a translatable one, once it exists).

Reply:
Oh, wasn't aware of that. Will update.